libs/model/src/models/chain_node.ts

```diff
@@ -43,6 +47,7 @@ export default (
       description: { type: dataTypes.TEXT, allowNull: true },
       ss58: { type: dataTypes.INTEGER, allowNull: true },
       bech32: { type: dataTypes.STRING, allowNull: true },
+      cosmos_gov_version: { type: dataTypes.STRING, allowNull: true },
```

Comment:
We might want to use an enum at the database level, but not a big deal since this field is totally controlled by us / can't have user-provided values.

Reply:
Yeah, it is enforced as an enum in code.